pod/ai_enhancement/templates/create_enhancement.html

```diff
@@ -0,0 +1,80 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block breadcrumbs %}
+    {{ block.super }}
+    <li class="breadcrumb-item">
+        <a
+            href="{% url 'video:video' slug=video.slug %}"
+            title="{% trans 'Access the video' %}"
+            aria-label="{{ video.title }}"
+        >
+            {{ video.title }}
+        </a>
+    </li>
+    <li class="breadcrumb-item active" aria-current="page">
+        {% trans 'Create enhancement' %}
+    </li>
+{% endblock breadcrumbs %}
+
+{% block page_content %}
+    <style>
+        .page_title {
+            display: none;
+        }
+    </style>
+    <h3>{% blocktrans with title=video.title %}Enhance the video <strong>{{ title }}</strong> with Aristote AI{% endblocktrans %}
+        <span class="float-end">
+            <a href="{% url 'video:video' slug=video.slug %}" title="{% trans 'Back to my video' %}" class="btn btn-secondary btn-sm" aria-label="{% trans 'Back to my video' %}">
+                <i class="bi bi-play-circle" aria-hidden="true"></i>&nbsp;{% trans 'Back to my video' %}
```

Comment:
Sur les autres pages du site, le contenu de la balise <a> est le suivant :
`<i class="bi bi-film" aria-hidden="true"></i>&nbsp;{% trans 'Back to the video' %}`

Reply:
Ça a été modifié